h/accounts/views.py

```diff
@@ -293,9 +285,11 @@ def reset_password_with_code_form(self):
         # If valid, we inject the supplied it into the form as a hidden field.
         # Otherwise, we 404.
         try:
-            user = schemas.ResetCode().deserialize(None, code)
+            user = schemas.ResetCode().deserialize(self.schema, code)
```

Comment:
I'm pretty sure passing in `None` for the colander node is fine. Is there a reason you're passing in the schema now?